src/libphonenumber/PhoneNumberUtil.php

```diff
@@ -679,6 +679,33 @@ public function loadMetadataFromFile($filePrefix, $regionCode, $countryCallingCo
         }
     }
 
+     /**
+     * Determines if a phone number is part of a special area code that bypasses logic
+     * @param $phoneNumber
+     * @return bool if true, the number is special and special action should be taken. False otherwise.
+     */
+    public static function areWeSpecialNumber($phoneNumber)
+    {
+        return preg_match("!\\+*[\\d]{10,11}!", $phoneNumber) > 0 ? true : false;
```

Comment:
I think that we need to add explicit checks for the area codes that we want to support.